packages/inter-protocol/test/auction/README-bidding-cli.md

```diff
@@ -0,0 +1,282 @@
+{
+  "id": "bid-1680241587424",
+  "payouts": {"Collateral":"3.105 ATOM"}
+}
+```
+
+See also [test-inter-cli.js.md](https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/test/snapshots/test-inter-cli.js.md) for exhaustive command options.
+
+## Installation
+
+To install `agd` and `agops` as well as prerequisite dev tools (`go`, `node`, `yarn`):
+
+```sh
+git clone https://github.com/Agoric/agoric-sdk
+cd agoric-sdk
+SKIP_DOWNLOAD=false ./bin/agd build
+```
+
+
+### Locating the `agd` command
+
+`agd` follows `go` conventions, so be sure `~/go/bin` is in your `$PATH`:
+
+```sh
+export PATH=~/go/bin:$PATH
+```
+
+Then try `agd version` to confirm.
+
+### Locating the `agops inter` command
+
+`agops` follows `npm`/`yarn` conventions, so you can `yarn run agops inter` as long as your current directory is under `agoric-sdk`.
+
+#### Add an alias for `agops inter`
+
+```sh
+alias inter="yarn run --silent agops inter"
+```
+
+Then you can just run...
+
+```
+$ inter
+Usage: agops inter [options] [command]
+
+Inter Protocol commands for liquidation bidding etc.
+...
+```
+
+
+## Key management with `agd`
+
+If you don't already have a testing address and corresponding private key:
+
+```sh
+agd keys add test-acct --keyring-backend=test
+```
+
+For example:
+
+```
+$ agd keys add test-acct --keyring-backend=test
+
+- name: test-acct
+  type: local
+  address: agoric18jr9nlvp300feu726y3v4n07ykfjwup3twnlyn
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"ApPNaneZjAWy2RarLaIw5O3vkBD6R3uZ9F2eRogei2BO"}'
+  mnemonic: ""
+
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+abandon plate bind ...
+```
+
+**CAUTION**: `agd keys add` displays the private mnemonic.
+
+
+Using the `test` keyring is lower friction, but feel free to use the os keyring:
+
+```sh
+agd keys add test-acct
+```
+
+
+## Adding some tokens to your account
+
+If you see something like this when you try sign and send a transaction with a brand new key...
+
+```
+$ inter bid by-price --from test-acct --give 10IST --price 3
+Error: key with address 3C8659F... not found: key not found
+Usage: ...
+...
+```
+
+It's because the blockchain has no record of your account until some tokens are sent to it.
+
+Use https://devnet.faucet.agoric.net/ to get some BLD and toy IBC asset tokens.
+
+Confirm by querying your balances:
+
+```
+$ agd query bank balances agoric18jr9nlvp... --node=https://devnet.rpc.agoric.net:443
+balances:
+- amount: "25000000"
+  denom: ibc/toyatom
+- amount: "75000000"
+  denom: ubld
+```
+
+**Note:** The URLs of the faucet, RPC node, explorer, etc. are available at https://devnet.agoric.net/ .
+
+## Provisioning your account for use with Zoe smart contracts
+
+Introducing your account to Zoe involves provisioning an offer handler.
+
+Again, use https://devnet.agoric.net/ but this time, choose "send IST and provision ...".
+
+
+## Liquidation Auction Status
```

Comment:
@dckc This feature isn't yet available to bidders right? It might confuse so thinking we either remove it or move it to bottom under a 'Coming Soon' section. Thoughts?

Reply:
#7266 landed, so it's available to bidders using code from master.

That code might not be deployed everywhere, but that's orthogonal to review of this PR.

Reply:
#7266 landed, so it's available to bidders using code from master.

That code might not be deployed everywhere, but that's orthogonal to review of this PR.

Reply:
ok so assumption is this will be available everywhere eventually - fine to leave it in then